_data/translation-management-systems.yaml

```diff
@@ -0,0 +1,669 @@
+
+    - service: prompsit
+
+    - service: smartmate
+
+    - service: tauyou
+
+    - service: tilde
+      custom: true
+
+    - service: yandex
+
+- id: memoq
+  name: MemoQ
+  description: Translation and localization management software
+  url: https://www.memoq.com
+  cat: true
+  documentation: https://docs.memoq.com/current/en/Welcome/welcome-memoq-docs.html
+  machine_translation_integrations:
+    - service: alexa-ai
+
+    - service: amazon
+      custom: true
+      glossary: true
+
+    - service: crosslang
+
+    - service: deepl
+      glossary: true
+      tone: true
+
+    - service: etranslation
+
+    - service: google
+      custom: true
+      glossary: true
+
+    - service: intento
+      glossary: true
+
+    - service: kantanmt
+      custom: true
+
+    - service: microsoft
+      custom: true
+
+    - service: mirai
+
+    - service: pangeamt
+      glossary: true
+
+    - service: systran
+      custom: true
+
+    - service: textra
+
+    - service: tilde
+      custom: true
+      glossary: true
+
+    - service: tmxmall
+
+  machine_translation_plugins:
+    - service: globalese
+
+  features:
+    - name: Local machine translation engine
+      description: Users can use their local machine translation in the tool
+
+    - name: Suggestions from multiple machine translation engines
+      description: Users can see segment-level suggestions from several machine translation systems at the same time
+
+    - name: Fuzzy repair with machine translation
+      description: Content that changes in the new source is fixed in target with machine translation
+
+- id: memsource
+  name: Memsource
+  description: Cloud-based translation management system and computer-assisted translation tool
+  url: https://www.memsource.com
+  cat: true
+  documentation: https://help.memsource.com/hc/en-us
+  machine_translation_integrations:
+    - service: aisa
+
+    - service: alexa-ai
+
+    - service: amazon
+      glossary: true
+
+    - service: apertium
+
+    - service: crosslang
+
+    - service: deepl
+      glossary: true
+
+    - service: fair-trade
+
+    - service: google
+      custom: true
+      glossary: true
+
+    - service: globalese
+
+    - service: human-science
+
+    - service: kantanmt
+      custom: true
+
+    - service: microsoft
+      custom: true
+      glossary: true
+
+    - service: mirai
+
+    - service: omniscien
+      custom: true
+
+    - service: pangeamt
+      custom: true
+
+    - service: prompsit
+
+    - service: promt
+      custom: true
+      glossary: true
+
+    - service: rozetta
+      custom: true
+      glossary: true
+
+    - service: language-weaver
+      custom: true
+      offline_glossary: true
+
+    - service: sundamt
+
+    - service: systran
+      custom: true
+
+    - service: tauyou
+
+    - service: tencent
+
+    - service: tilde
+      custom: true
+
+    - service: ubiqus
+      glossary: true
+
+    - service: yandex
+
+  quality_integrations:
```

Comment:
Is this about QE?  I don't think that their QE feature is called Memsource Translate, if it has a name.

Reply:
Yes, it is. I should rename it to `quality_estimation`.
I guess I could make `quality_integrations` a boolean and, if the service is named, then add `name`.

Reply:
Boolean has problems.  I'd just call it *Memsource quality estimation*

Reply:
Later we'll make a list of those too.

Reply:
I put `memsource-mtqe` so that it's the ID.
I'll add it to `quality.yml` so that refers back to that file.